doc/ale.txt

```diff
@@ -1142,6 +1142,7 @@ ALEPrevious                                                       *ALEPrevious*
 ALEPreviousWrap                                               *ALEPreviousWrap*
 ALENext                                                               *ALENext*
 ALENextWrap                                                       *ALENextWrap*
+ALENextWrap                                                       *ALENextWrap*
```

Comment:
Thanks @polyzen, I removed the duplicate. The actual documentation for ALEDetail is below.